### PR TITLE
Update repo ReadMe.md file

### LINKED ISSUE #21: Update repo ReadMe.md file
In line with the issue available [here](https://github.com/BHoM/admin/issues/4), please update your repositories ReadMe.md file to match the template provided in the linked admin issue.

Please remember to change any instances of `Robot_Toolkit` in the template to your repository name.

I will comment separately with a specific AppVeyor badge link that should be used for this repo.

When you resolve this issue, please add myself and @al-fisher as reviewers.

### PR DESCRIPTION
**Pull request by adecler**
### Issues addressed by this PR

Closes #21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,27 @@
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/my3nfqs186bkh325/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/csharp_toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/CSharp_Toolkit/CSharp_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=9176&branchName=master)
```
Comment (FraserGreenroyd) [inline README.md]: Azure badge should be `[Build Status](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=174&branchName=master)`

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Comment by FraserGreenroyd** [comment]
Also not sure about the CI checks on AppVeyor. They're not happy but this PR doesn't change anything functional with the code. I would say it warrants investigation, but I'm not thinking this PR is the scope to fix whatever they're complaining about given this PR is about the ReadMe only.

**Review by FraserGreenroyd — APPROVED** [review]

**Comment by FraserGreenroyd** [comment]
I've raised [this issue](https://github.com/BHoM/CSharp_Toolkit/issues/23) to investigate the CI issue, this PR can be merged by someone with merge rights based on the approval above :smile:

**Comment by al-fisher** [comment]
Yep happy to merge as this is an isolated AppVeyor issue - and not this PR

**Pull request merged**